[2025-01-24T20:05:28.951+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T20:05:28.967+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-24T20:05:28.977+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-22T00:00:00+00:00 [queued]>
[2025-01-24T20:05:28.978+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2025-01-24T20:05:29.001+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-22 00:00:00+00:00
[2025-01-24T20:05:29.026+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=815) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T20:05:29.028+0000] {standard_task_runner.py:72} INFO - Started process 853 to run task
[2025-01-24T20:05:29.028+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'scheduled__2025-01-22T00:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpb7yvgs6m']
[2025-01-24T20:05:29.031+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask init_folders_task
[2025-01-24T20:05:29.081+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task scheduled__2025-01-22T00:00:00+00:00 [running]> on host 972b45cbfb12
[2025-01-24T20:05:29.161+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-22T00:00:00+00:00'
[2025-01-24T20:05:29.163+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T20:05:29.192+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T20:05:29.193+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=scheduled__2025-01-22T00:00:00+00:00, execution_date=20250122T000000, start_date=20250124T200528, end_date=20250124T200529
[2025-01-24T20:05:29.244+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T20:05:29.278+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T20:05:29.281+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
